src/pages/SinglePracticePage/styled.ts

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+export const SinglePracticeWrapper = styled.section``;
+
+export const Heading = styled.h2`
+  margin-left: 100px;
+`;
+
+export const Space = styled.div`
```

Comment:
is it possible to make use of css selectors instead to add spacing? Might not be that easy from how i understand styled-components work though, but it would help to prevent the DOM from getting polluted with empty containers if possible

Reply:
I've now removed the Space component, and added css classNames to the other components, to make it easier to select them from the SinglePracticeWrapper, and then add the spacing from there